### ISSUE TITLE
Add keyboard shortcut for "Rollback"

### ISSUE THREAD

**Issue by LuckyTil** [laurentkempe/GitDiffMargin#350]
Add keyboard shortcut for "Rollback", please.

**Comment by laurentkempe** [comment]
@LuckyTil At the moment, you can have a shortcut to open the popup and then navigate the option with the arrow keys.

Cheers
Laurent

**Comment by eightbit11** [comment]
> @LuckyTil At the moment, you can have a shortcut to open the popup and then navigate the option with the arrow keys.
> 
> Cheers
> Laurent

It would be very convenient to have a shortcut key.

**Comment by laurentkempe** [comment]
> It would be very convenient to have a shortcut key.

I agree, it is just a question of time, you can also make a PR @eightbit11
